webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/queries/DynGroupService.scala

```diff
@@ -106,6 +123,61 @@ class DynGroupServiceImpl(
       dyngroupIds.sortBy(numberOfQuery)
     }
   }.toBox
+
+  override def changesSince(lastTime: DateTime): Box[Boolean] = {
+    val n0 = System.currentTimeMillis
+    if(n0 < lastTime.getMillis || n0 - lastTime.getMillis < 100) {
```

Comment:
does that mean a change in the future?

Reply:
ah yes, it's not usefull in fact (because if the first is true, the second is also). I will remove it.